MdeModulePkg/MdeModulePkg.dec

```diff
@@ -1454,7 +1454,8 @@
   #   TRUE - VariablePolicy can be disabled by request through the interface (until interface is locked)
   #   FALSE - VariablePolicy interface will not accept requests to disable and is ALWAYS ON
   # @Prompt Allow VariablePolicy enforcement to be disabled.
-  gEfiMdeModulePkgTokenSpaceGuid.PcdAllowVariablePolicyEnforcementDisable|FALSE|BOOLEAN|0x30000020
+  ## MU_CHANGE - Allow policy to be disabled by default.
+  gEfiMdeModulePkgTokenSpaceGuid.PcdAllowVariablePolicyEnforcementDisable|TRUE|BOOLEAN|0x30000020

```

Comment:
Is this one where we should want to default FALSE behavior now? @makubacki

Reply:
I'd also prefer that to be the default behavior. We should check existing usage before doing so. If everyone just sets the PCD to TRUE in their DSC, we haven't changed anything and caused churn. I filed this issue to follow up https://github.com/microsoft/mu_basecore/issues/1050. I suggest we use that and leave it as-is for now.